src/condor_utils/ipv6_addrinfo.cpp

```diff
@@ -300,11 +66,11 @@ double getaddrinfo_slow_limit = 2.0;
 void (*getaddrinfo_slow_callback)(const char *node, const char *service, double timediff) = NULL;
 
 int ipv6_getaddrinfo(const char *node, const char *service,
-		addrinfo_iterator& ai, const addrinfo& hint)
+		addrinfo*& ai, const addrinfo& hint)
 {
-	addrinfo* res = NULL;
+	ai = nullptr;
```

Comment:
Isn't usually bad form to unconditionally modify an output parameter?

Reply:
Fair point.